test/ct-e2e.yaml

```diff
@@ -0,0 +1,38 @@
+debug: true
+target-branch: dev
+chart-dirs:
+  - stable
+  - staging
+excluded-charts:
+  - common
+  - dex-controller # Moved to a different helm repo
+  - azuredisk-csi-driver  # DCOS-62804
```

Comment:
Its my hope that once all the tickets are closed we can remove the special *e2e* variant of the config and both linting and install and upgrade tests can share just the one config.